### PR TITLE
refactor: create application-tasklist.properties

### PR DESCRIPTION
**Pull request by houssain-barouni**
## Description
* Replace `application-tasklist.yml` by `application-tasklist.properties`
* Move properties defined in `StandaloneTasklist` to `application-tasklist.properties`
* Make integration tests use `application-tasklist.properties` (using `tasklist` spring profile), to better test and reflect the configuration used in production
 
## Related issues

relates to https://github.com/camunda/zeebe/issues/18535

### CODE REVIEW

File: dist/src/main/resources/application-tasklist.properties [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+camunda.identity.audience=${camunda.tasklist.identity.audience:}
+camunda.identity.baseUrl=${camunda.tasklist.identity.baseUrl:}
+
+#---
+spring.config.activate.on-profile=sso-auth
+camunda.identity.type=AUTH0
+camunda.identity.baseUrl=${camunda.tasklist.identity.baseUrl:${CAMUNDA_TASKLIST_IDENTITY_BASEURL:}}
+camunda.identity.issuer=${camunda.tasklist.identity.issuerUrl:${CAMUNDA_TASKLIST_IDENTITY_ISSUER_URL:}}
+camunda.identity.issuerBackendUrl=${camunda.tasklist.identity.issuerBackendUrl:${CAMUNDA_TASKLIST_IDENTITY_ISSUER_BACKEND_URL:}}
+camunda.identity.audience=${camunda.tasklist.identity.audience:}
```
Comment (nathansandi) [inline dist/src/main/resources/application-tasklist.properties:43]: You can delete this line (43), So far after analyse, it wont be used.

File: tasklist/qa/integration-tests/src/test/java/io/camunda/tasklist/management/HealthCheckAuthenticationIT.java [hunk 2]
```diff
@@ -55,8 +54,7 @@
       ElasticsearchConnector.class
     },
     webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-@ContextConfiguration(initializers = AddManagementPropertiesInitializer.class)
-@ActiveProfiles(AUTH_PROFILE)
+@ActiveProfiles({AUTH_PROFILE, "tasklist", "test"})
```
Comment (nathansandi) [inline tasklist/qa/integration-tests/src/test/java/io/camunda/tasklist/management/HealthCheckAuthenticationIT.java:57]: [Question] Why we are using 3 active profiles over here?

Comment (houssain-barouni) [reply tasklist/qa/integration-tests/src/test/java/io/camunda/tasklist/management/HealthCheckAuthenticationIT.java:57]: - `AUTH_PROFILE` is required by this test scenario 
- `tasklist` is to import the properties in `application-tasklist.properties` (in `dist` folder), the goal is to test similar configuration to the one that is used by the distribution
- `test` is to import the properties in `applicaiton-test.yml` (some specific properties that is used for tests like fake static files...)

File: dist/src/main/resources/application-tasklist.properties (hunk 1 above)
Comment (houssain-barouni) [reply dist/src/main/resources/application-tasklist.properties:43]: Thanks, I will remove it

Comment (houssain-barouni) [reply dist/src/main/resources/application-tasklist.properties]: Fixed

File: tasklist/qa/integration-tests/src/test/java/io/camunda/tasklist/management/HealthCheckAuthenticationIT.java (hunk 2 above)
Comment (nathansandi) [reply tasklist/qa/integration-tests/src/test/java/io/camunda/tasklist/management/HealthCheckAuthenticationIT.java:57]: Thanks for clarifying this. 💯

**Review by nathansandi — APPROVED** [review]

**Pull request merged**